etc/tar.profile

```diff
@@ -18,6 +18,7 @@ notv
 novideo
 shell none
 tracelog
+caps.drop all
```

Comment:
are the capabilities not required for setting certain file flags, owner etc.?

Reply:
removed caps.drop all